pkg/storaged/storage-controls.jsx

```diff
@@ -117,7 +117,8 @@ export class StorageButton extends React.Component {
                             content={(excuse) => (
                                 <button id={this.props.id}
                                             onClick={checked(this.props.onClick)}
-                                            className={classes + (excuse ? " disabled" : "")}>
+                                            className={classes + (excuse ? " disabled" : "")}
+                                            disabled={excuse}>
```

Comment:
I suppose `excuse` is a string. In this case, this  should be `disabled={!!excuse}` to be a boolean value?

Reply:
In es6 at least the empty strings are falsy whilst every string of lengh > 0 is truthy. So this should work.